bpf/lib/common.h

```diff
@@ -664,6 +668,18 @@ enum metric_dir {
 #define DSR_IPV6_OPT_LEN	(sizeof(struct dsr_opt_v6) - 4)
 #define DSR_IPV6_EXT_LEN	((sizeof(struct dsr_opt_v6) - 8) / 8)
 
+/* Geneve option used to carry service addr and port for DSR.
+ *
+ * Class = 0x200 (unassigned according to [1])
+ * Type  = 0x20  (vendor-specific)
+ *
+ * [1]: https://www.iana.org/assignments/nvo3/nvo3.xhtml#geneve-option-class
+ */
+#define DSR_GENEVE_OPT_CLASS	0x200
```

Comment:
I've applied for a Geneve Option Class for the Cilium project via IANA. Let's hold back on merging this until we hear back.

Reply:
Awesome! I will update it when we hear from them

Reply:
We have been allocated 0x014B: https://www.iana.org/assignments/nvo3/nvo3.xhtml

Reply:
Thanks! I will use it.